vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -148,8 +148,6 @@ public DateTimePicker(LocalDateTime initialDateTime) {
         addToSlot(datePicker, "date-picker");
         addToSlot(timePicker, "time-picker");
 
-        setLocale(UI.getCurrent().getLocale());
```

Comment:
Setting the custom locale field to UI locale is not necessary anymore, DatePicker and TimePicker will automatically use the current UI locale internally.